logbook-jmh/README.md

```diff
@@ -0,0 +1,42 @@
+# logbook-jmh
```

Comment:
Would the same setup allow us to compare a simple web app with and without logbook to compare IO throughput/latency?

Reply:
I'm not sure, I've used other tools, like Gatling, for that; it is just not so 'micro' anymore. Ideally you'll want the test driver and the system under test on two systems to be completely fair, the driver will consume considerable resources. 

But I am guessing this can be done, at least for determining the introduced delay when not in saturation.

Reply:
I think I'm going to go with 'no'. What you'd like is a reference point for the current implementation, and then interpolate as performance changes. You'll probably see with and without logging is something like 10-25% worth (loosely based on [this](https://www.greenbird.com/2015/06/11/logging-soap-traffic-in-mule-esb-versions-3-2-to-3-6)).